src/main/java/com/makhnov/springbootcalculator/dao/CalculatorDao.java

```diff
@@ -0,0 +1,40 @@
+package com.makhnov.springbootcalculator.dao;
+
+import com.makhnov.springbootcalculator.PostfixResult;
+import org.jooq.*;
+import org.jooq.impl.DSL;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+
+import static org.jooq.impl.DSL.field;
+import static org.jooq.impl.DSL.name;
+import static org.jooq.impl.DSL.table;
+
+@Repository
+public class CalculatorDao {
+    @Qualifier("dataSource")
+    @Autowired
+    private DataSource dataSource;
+
+    public void saveResult(String string) throws Exception {
+        PostfixResult postfixResult = new PostfixResult();
+        Connection connection = dataSource.getConnection();
+        DSLContext dslContext = DSL.using(connection, SQLDialect.MYSQL);
```

Comment:
it's wrong
we already discussed about it
1) create data a new data source https://stackoverflow.com/questions/28821521/configure-datasource-programmatically-in-spring-boot
2) create DSLContext as a new bean